packages/xstate-react/src/useActor.ts

```diff
@@ -52,7 +52,17 @@ export function useActor(
   const deferredEventsRef = useRef<EventObject[]>([]);
   const [current, setCurrent] = useState(() => getSnapshot(actorRef));
 
-  const send: Sender<EventObject> = useConstant(() => (event) => {
+  const send: Sender<EventObject> = useConstant(() => (...args) => {
+    const event = args[0];
+
+    if (process.env.NODE_ENV !== 'production' && args.length > 1) {
+      console.warn(
+        `Unexpected argument: ${JSON.stringify(
+          (args as any)[1]
+        )}. Only a single event object can be sent to actor send() functions.`
+      );
```

Comment:
Especially the "Unexpected argument" part sounds like something that would be included in a hard error whereas this, for now, is just a deprecation warning. Maybe a little bit more mild wording could be used here?

Reply:
```suggestion
      console.warn(
        `Unexpected payload: ${JSON.stringify(
          (args as any)[1]
        )}. Only a single event object can be sent to actor send() functions.`
      );
```